lib/evm/eei.ts

```diff
@@ -11,13 +11,15 @@ export default class EEI {
   _result: any
   _state: PStateManager
   _interpreter: any
+  _lastReturned: Buffer
 
   constructor (env: any, runState: any, result: any, state: PStateManager, interpreter: any) {
     this._env = env
     this._runState = runState
     this._result = result
     this._state = state
     this._interpreter = interpreter
+    this._lastReturned = Buffer.alloc(0)
```

Comment:
Also just for understanding: what is the main reasoning for this move?

Reply:
The main reasoning was to localize variables as much as possible. `lastReturned` is there to track the last value returned from a call/create. It was only used in `EEI`, so I moved it from `runState`. But if the call/create logic moves elsewhere (I'm thinking `Interpreter`) this might need to be moved again. So, not quite sure yet where to put it.

Reply:
Ah ok, makes sense.